svc.go

```diff
@@ -24,8 +27,11 @@ const P_CIRCUIT = 290
 var (
 	AutoNATServiceDialTimeout   = 15 * time.Second
 	AutoNATServiceResetInterval = 1 * time.Minute
+	AutoNATServiceResetJitter   = 15 * time.Second
 
-	AutoNATServiceThrottle = 3
+	AutoNATServiceThrottle  = 3
+	AutoNATGlobalThrottle   = 30
+	AutoNATMaxPeerAddresses = 4
```

Comment:
https://github.com/libp2p/go-libp2p-autonat-svc/issues/39 indicated this number should be 16. Was there a discussion/update indicating that this should be 4?